avocado_vt/loader.py

```diff
@@ -32,6 +31,12 @@
 from .discovery import DiscoveryMixIn
 from .test import VirtTest
 
+try:
+    from avocado.core import loader
+    AVOCADO_LOADER_AVAILABLE = True
+except ImportError:
+    AVOCADO_LOADER_AVAILABLE = False
+
```

Comment:
I would like to suggest creating `loade.register_plugin` method and move this there, so you could do the import only on one place and not having it in every file where you want to use loader. Something similar is done in [virttest/compat.py](https://github.com/avocado-framework/avocado-vt/blob/master/virttest/compat.py). What do you think?

Reply:
I thought about that, but here's a huge chunk of code in `avocado_vt/loader.py` (`class VirtTestLoader(loader.TestLoader, DiscoveryMixIn)` that can not be evaluated if there is no `avocado.core.loader` available. That is, it's not only about `loader.register_plugin` working in two different ways (which is what most of `virttest/compat.py` does).

If you think this is a must, I can try to rethink a bit more what can be done, but I believe the best solution would be to open an issue so that the conditional here can be dropped when 92LTS is EOL'ed.

What do you think?

Reply:
Sure, I don't have a problem with your solution.